CHANGELOG.md

```diff
@@ -23,10 +23,23 @@ title: Changelog
 
 ## Table of Contents
 
+- [0.4.0](#040)
 - [0.3.0](#030)
 - [0.2.0](#020)
 - [0.1.0](#010)
 
+## 0.4.0
+
+### Feature
+
+- :sunrise: feat: add response-rewrite plugin [#91](https://github.com/apache/apisix-go-plugin-runner/pull/91)
+- :sunrise: feat: support response filter for plugin [#90](https://github.com/apache/apisix-go-plugin-runner/pull/90)
+- :sunrise: feat: add debugf function [#87](https://github.com/apache/apisix-go-plugin-runner/pull/87)
+
+### Change
+
+- change: rename http.Response to http.ReqResponse  [#88](https://github.com/apache/apisix-go-plugin-runner/pull/88)
```

Comment:
https://github.com/apache/apisix-go-plugin-runner/pull/88 isn't a change PR?

Reply:
We need to describe a break change that we require the Plugin to embed a default implementation.